protobuf_definitions/telemetry.proto

```diff
@@ -107,22 +107,8 @@ message ErrorFlagsTel {
 }
 
 /**
-  * Receive the current state of the auto heading controller.
+  * Receive the current state of the control system.
   */
-message AutoHeadingTel {
-  AutoHeadingState state = 1; // State of the heading controller
+message CotrolModeTel {
```

Comment:
@aviggen  "Cotrol" is a spelling mistake. And make sure to update in p2_drone as well.